Story_Builder/BulkSystem.md

```diff
@@ -0,0 +1,1307 @@
+simplified ways. These models prohibit character actions that should not
+be possible. Without such prohibitions players\' suspension of disbelief
+would be broken.
+
+Most games have a model for bulk. These models might have a variety of
+results from limiting item storage (i.e., limiting character inventory)
+to modifying character abilities (i.e., reducing stamina).
+
+Some systems also model dimensionality, allowing objects to have a size.
+These models determine whether items can fit in containers (or rooms) or
+describe the size of items relative to one another.
+
+### Approaches
+
+Many multiplayer interactive fiction systems take a minimalist approach
+to modeling the physical world.
+
+In the simplest model of bulk, items are given a mass and containers are
+given a maximum mass. This is easy for players to understand. However,
+differing densities cause paradoxes such as a feather pillow taking up
+the same amount of space in a container as a gold coin.
+
+Most systems extend this model by giving items a size (volume) and
+containers a maximum volume. Again, this is easy for players to
+understand. It also doesn\'t require much extra work by the
+Admins.StoryBuilder. However inconsistencies still crop up that undercut
+the realism of the simulated world.
+
+A few systems have drastically increased the number of physical
+attributes for items. This results in more realistic physics but also
+produces a very complex model. Admins.StoryBuilders must put more effort
+into each item. Inconsistencies and errors are common.
+
+### Problems Addressed by the Skotos Bulk System
+
+A chatroom wouldn\'t require a complex bulk system. However in a fully
+featured game players will inevitably encounter complex situations
+involving bulk. Currently the Skotos system does this by measuring bulk
+by two core values: mass and volume (the latter via a density
+calculation).
+
+This allows the Skotos Bulk System to model tricky situations such as:
+
+* A large winged dragon tries to enter a small houses. The dragon would
+not fit inside the house. 
+* Multiple swords being shoved into a scabbard. A scabbard would be designed to fit a sword.
+
+
+### Problems NOT Addressed
+
+There are desirable features that we have not addressed in the Bulk
+System at this time. Some of these features we actually implemented in
+the past, but removed for performance reasons \-- others we felt added
+too much complexity.
+
+* The large winged dragon tries to enter a very large house through a
+very small door. Ideally the dragon would not fit through a small door
+(without destroying it and, possibly, the house). 
+* People of different
+size contesting for seating room. A particular oak bench might be able
+to seat two ogres, four humans, or six dwarves. 
+* Objects being pushed
+through small holes. A rope could be fed through a small hole into a
+large cave. Sadly an elephant would not fit through the same hole.
+
+[QuickSummaryBulkSystem]
```

Comment:
I don't think a MixedCase-in-square-brackets link like this will actually link in GitHub Markdown. You'll probably need to do the square-brackets-then-parens thing.